packages/orchestra/lib/client.ts

```diff
@@ -0,0 +1,84 @@
+import type { Json } from '@nangohq/scheduler';
+import { route as scheduleRoute } from './routes/v1/schedule.js';
+import { route as outputRoute } from './routes/v1/taskId/output.js';
+import type { Result } from '@nangohq/utils';
+import { Ok, Err, routeFetch } from '@nangohq/utils';
+
+interface SchedulingProps {
+    name: string;
+    groupKey: string;
+    retry: {
+        count: number;
+        max: number;
+    };
+    timeoutsInSecs: {
```

Comment:
This property name for this object is confusing. 

Maybe `timeoutSettingsInSeconds` ?